demo/tsdash-demo-driver.js

```diff
@@ -65,6 +65,7 @@ TsDashboardDemoDriver.prototype.getDrawData = function (options, callback) {
     var ts3_curr = 3;
     var nodes = {};
     var edges = [];
+    var alerts = [{"type": "db", "ts":1476704004495}, {"type": "events", "ts":1476704004535}, {"type": "events", "ts":1476704004695}, {"type": "blalalalaaaa", "ts":1476704004995}];
```

Comment:
Rename this to something more neutral - like "events" with some non-computer-related types